src/dfx-core/src/build.rs

```diff
@@ -0,0 +1,35 @@
+use std::{env, fs::File, io::Write, path::Path};
+
+const CANISTER_IDS_URL: &str = "https://raw.githubusercontent.com/dfinity/ic/1402bf35308ec9bd87356c26f7c430f49b49423a/rs/nns/canister_ids.json";
```

Comment:
It looks like we only add a well-known canister once every year or two.  Let's instead copy the file into our repo rather than downloading it on each build.

Reply:
Addition of a "well-known" canister is infrequent enough that I think it would be ok to leave it to manual update. If you feel inspired to update it in `update-replica-version.yaml` that's fine too.

Reply:
Alright, I can agree with that. Lets leave it manual for now then. If automation is needed you can always reach out and I would be happy to add it.